src/Websitetoolboxcommunity.php

```diff
@@ -219,9 +217,18 @@ public function afterSaveSettings(): void{
         $loggedinUserId       = Craft::$app->getUser()->getIdentity()->id;
         $loggediUserName     = Craft::$app->getUser()->getIdentity()->username;
         $postData     = array('type'=>'json','apikey' => $result->forumApiKey, 'user' => $loggediUserName,'email'=>$loggedinUserEmail,'externalUserid'=>$loggedinUserId);
-        $response       = Websitetoolboxcommunity::getInstance()->sso->sendApiRequest('POST',$RequestUrl,$postData,'json');         
-        setcookie("forumLogoutToken", $response->authtoken, time() + 3600,"/");
-        setcookie("forumLoginUserid", $response->userid, time() + 3600,"/");         
+        $response       = Websitetoolboxcommunity::getInstance()->sso->sendApiRequest('POST',$RequestUrl,$postData,'json');
+        if(isset($response->authtoken) && $response->authtoken !=''){
+            setcookie("forumLogInToken", $response->authtoken, time() + 3600,"/");
+            setcookie("forumLogoutToken", $response->authtoken, time() + 3600,"/");
+            setcookie("forumLoginUserid", $response->userid, time() + 3600,"/");    
+        }else{
+            $errorMessage = 'Website Toolbox Community error, user is not logged in.';
```

Comment:
Why you show hard coded message. We already provide error message, not need to show message menaully.